modules/migration-adding-cluster-to-mtc.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// migration/migrating_3_4/migrating-applications-with-cam.adoc
-// migration/migrating_4_1_4/migrating-applications-with-cam.adoc
-// migration/migrating_4_2_4/migrating-applications-with-cam.adoc
-[id='migration-adding-cluster-to-cam_{context}']
-= Adding a cluster to the CAM web console
+// migration/migrating_3_4/migrating-applications-with-mtc.adoc
+// migration/migrating_4_1_4/migrating-applications-with-mtc.adoc
+// migration/migrating_4_2_4/migrating-applications-with-mtc.adoc
+[id='migration-adding-cluster-to-mtc_{context}']
+= Adding a cluster to the {mtc-short} web console
 
-You can add a source cluster to the CAM web console.
+You can add a source cluster to the {mtc-short} web console.
 
 .Prerequisites for Azure
 
```

Comment:
In the preview, the Prerequisites for Azure are a paragraph with sentences separated by asterisks, not a bulleted list,

Reply:
I see a bullet list